pkg/scheduler/ugm/user_tracker.go

```diff
@@ -97,6 +109,9 @@ func (ut *UserTracker) hasGroupForApp(applicationID string) bool {
 func (ut *UserTracker) setGroupForApp(applicationID string, groupTrack *GroupTracker) {
 	ut.Lock()
 	defer ut.Unlock()
+	if groupTrack != nil {
+		ut.events.sendAppGroupLinked(applicationID, groupTrack.groupName)
```

Comment:
the first parameter should be `group`, and second is `applicationID`

Reply:
Good catch... extended the test to validate this